packages/transitive-overlay/src/index.tsx

```diff
@@ -139,7 +143,10 @@ const TransitiveCanvasOverlay = ({
                       type: "Feature",
                       properties: {
                         type: "street-edge",
-                        color: modeColorMap[segment.type] || "#008",
+                        color:
+                          defaultColorOverride ||
```

Comment:
Sorry last nitpick -- can we rename this just `colorOverride`? Since it now overrides every possible color